service/src/main/resources/application.properties

```diff
@@ -62,6 +62,7 @@ fs.defaultFS=
 elasticsearch.host=localhost
 elasticsearch.port=9200
 elasticsearch.scheme=http
+index=griffin
```

Comment:
The property name "elasticsearch.index" fits better in my opinion.

Reply:
elasticsearch.index make sense to me.